app/results.py

```diff
@@ -77,11 +77,8 @@ def write_result(self, job_id, result):
         :rtype: boolean
         """
         try:
-            # base64 encode the result before encrypting to ensure compat
-            encodedResult = base64.b64encode(result)
-
-            # encrypt the encoded result
-            encryptedResult = self.aescipher.encrypt(encodedResult)
+            # encrypt the encoded result and then encode it
+            encryptedResult = base64.b64encode(self.aescipher.encrypt(result))
```

Comment:
I'm curious as to why we are doing this?

Reply:
After you encrypt the bytes, we want to base64 encode the result. It is helpful later when we consolidate. We could encode before we append to the consolidated result too.